### PR TITLE
Initial setup.

### PR DESCRIPTION
**Pull request by KaleabMekonen**
This is a setup commit which contains the basic structure of the codebase.

### CODE REVIEW

File: Initial Setup/Initial Setup.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,286 @@
+// !$*UTF8*$!
```
Comment (KaleabMekonen) [inline Initial Setup/Initial Setup.xcodeproj/project.pbxproj]: Add .ignore file for **/**.xcodeproj

**Comment by tomhydra** [comment]
I like the setup 👍  Why did you want to use `.rtf` format instead of `.md` file for problem descriptions though? Using .md file will make it easy to also view it on github.

**Comment by KaleabMekonen** [comment]
> I like the setup 👍 Why did you want to use `.rtf` format instead of `.md` file for problem descriptions though? Using .md file will make it easy to also view it on github.

**Comment by KaleabMekonen** [comment]
> I like the setup 👍 Why did you want to use `.rtf` format instead of `.md` file for problem descriptions though? Using .md file will make it easy to also view it on github.

I changed it back to .md file

**Pull request merged**